### DISCUSSION TITLE
CUSA07511 - Hellblade Senua’s Sacrifice

### DISCUSSION THREAD

**Discussion by gorganps4** [GoldHEN/GoldHEN_Cheat_Repository#7] [Cheat requests]
Please add CUSA07511 - Hellblade Senua’s Sacrifice

**Comment by kmeps4** [comment]
Will check if there's a cheat available to add

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
